ldk/javascript/examples/self-test-loop/src/tests/whisper/index.ts

```diff
@@ -1901,15 +1901,49 @@ export const testAlignItems = (): Promise<boolean> =>
AAAAAAAAAAAAAAAAAAAFM2AmjmRpnmiqrmzrvjBLOY/1UgcAJJfbEAaEIOIrMBYBiYuCGAQUveXjEY1Zr9isdmsKAQAh+QQAAwAAACwAAAAAFAAUAIRiGv9jHP9jHf9nIf9oI/9tK/9xMf94Ov9+Q/9/Rf+DSv+ETP+FTv+JU/+pgv+0kv+3lv+4mP+6mv+7nf+8nf/k2P/l2f/m2v/n3f/o3v/p3/////8AAAAAAAAAAAAAAAAFNOAmjmRpnmiqrmzrvjBrQRH2WggAKJr7FIfFgOIzOBoBYsuSIAQWPZclEonGrtisdss1hQAAIfkEAAMAAAAsAAAAABQAFACEYhr/Yhv/Yxz/ZB3/aCP/aST/bSv/cTH/eDr/fkP/f0X/g0r/hEz/ilT/qYL/tJL/t5b/uJj/upr/u5z/vJ3/5Nj/5dn/5tr/593/6N7/6d//////AAAAAAAAAAAAAAAABTPgJo5kaZ5oqq5s674wa0ER9loJACyZ+xiIRmHiOzgaA4rLoiAIGL1lJBKNWa/YrHZrCgEAIfkEAAMAAAAsAAAAABQAFACEYhr/Yhv/Yxz/ZB3/aCP/aif/bSv/cTH/eDr/fkP/f0X/g0v/hEz/ilT/jVn/qYL/tJL/t5b/uJj/upr/u5z/vJ3/5Nj/5dn/5tr/593/6N7/6d//////AAAAAAAAAAAABTMgJ45kaZ5oqq5s674we0VS9l4JACyaCxkIR4HiOzwag4rroiAIGL2lRBKNWa/YrHZrCgEAIfkEAAMAAAAsAAAAABQAFACEYhr/Yxz/ZB3/aCP/bSr/bSv/cTH/dzr/fkP/f0X/g0v/hEz/ilX/j13/qYL/tJL/tpb/uJj/upr/u5z/vJ3/5Nj/5dn/5tr/593/6N7/6d//////AAAAAAAAAAAAAAAABTLgJo5kaZ5oqq5s674wa0ER9loIACyZ+xSHRmHiMzgYAorLkhgEeLdIpBerWq/YrPYUAgAh+QQAAwAAACwAAAAAFAAUAIRiGv9jHP9kHf9oI/9tK/9uLP9xMf93Ov9+Q/9/Rf+ES/+ETP+KVf+RX/+pgv+0kv+3lv+4mP+6mv+7nP+8nf/k2P/l2f/m2v/n3f/o3v/p3/////8AAAAAAAAAAAAAAAAFM+AmjmRpnmiqrmzrvjBrQRH2WggQKJn7EIdGYeIzOBgCisuSGAQWvWUkEo1Zr9isdmsKAQAh+QQAAwAAACwAAAAAFAAUAIRiGv9jHP9jHf9kHf9oI/9tK/9wL/9xMf93Of9+Q/9/Rf+ETP+KVf+TYf+pgv+0kv+3lv+4mP+6mv+7nP+8nf/k2P/l2f/m2v/n3f/o3v/p3/////8AAAAAAAAAAAAAAAAFMuAmjmRpnmiqrmzrvjBrQRH2WgkQLJn7FIiGYeI7OBgDisuiIOx6y0gEGqtar9is1hQCACH5BAADAAAALAAAAAAUABQAhGIa/2Mc/2Qd/2gj/20r/3Ex/3c5/35D/39F/4RM/4RN/4pV/5Nj/6mC/7SS/7eW/7iY/7qa/7uc/7yd/+TY/+XZ/+ba/+fd/+je/+nf/////wAAAAAAAAAAAAAAAAAAAAUyoCaOZGmeaKqubOu+MFs90PVWByAkmOsQBkZB4is0FoKJq4IYBHg3CKQXq1qv2Kz2FAIAIfkEAAMAAAAsAAAAABQAFACEYhr/Yxz/ZB3/ZB7/aCP/bSv/cTH/czP/dzn/fkP/f0X/hEz/hU3/ilX/lGT/qYL/tJL/t5b/uJj/upr/u5z/vJ3/5Nj/5dn/5tr/593/6N7/6d//////AAAAAAAAAAAABTIgJ45kaZ5oqq5s674we0VS9l4JICybCxUIh4HiMzwagorroiAEeDeJpBerWq/YrPYUAgAh+QQAAwAAACwAAAAAFAAUAIRiGv9jHP9kHf9kHv9oI/9tK/9xMf90NP93Of9+Q/9/Rf+ETP+FTv+KVf+VZf+pgv+0kv+2lv+4mP+6mv+7m/+8nf/k2P/l2f/m2v/n3P/o3v/p3/////8AAAAAAAAAAAAFMiAnjmRpnmiqrmzrvjB7RVL2XgkgLJsLFQjHgeIzPBqCiuuiIAR4N4mkF6tar9is9hQCACH5BAADAAAALAAAAAAUABQAhGIa/2Mc/2Qd/2Qe/2gj/20r/3Ex/3U2/3c5/35D/39F/4RM/4VO/4pU/5Vl/6mC/7SS/7eW/7iY/7qa/7ub/7yd/+TY/+XZ/+ba/+fc/+je/+nf/////wAAAAAAAAAAAAUyICeOZGmeaKqubOu+MHtFUvZeCSAsmwsVCMeh4jM8GgJi66IgBHg3iaQXq1qv2Kz2FAIAIfkEAAMAAAAsAAAAABQAFACEYhr/Yxz/ZB3/ZR7/aCP/bSv/cTH/dTf/dzn/fkP/f0X/hEz/hk7/ilX/lmb/qYL/tJL/t5b/uJj/upr/upv/vJ3/5Nj/5dn/5tr/59z/6N7/6d//////AAAAAAAAAAAABTIgJ45kaZ5oqq5s674we0VS9l4JICybCxUIx6HiMzwaAmLroiAEeDeJpBerWq/YrPYUAgAh+QQAAwAAACwAAAAAFAAUAIRiGv9jHP9kHf9lHv9oI/9tK/9xMf91N/93Of9+Q/9/Rf+ETP+GTv+KVf+WZv+pgv+0kv+3lv+4mP+6mv+6m/+8nf/k2P/l2f/m2v/n3P/o3v/p3/////8AAAAAAAAAAAAFMiAnjmRpnmiqrmzrvjB7RVL2XgkgLJsLFQjHoeIzPBoCYuuiIAR4N4mkF6tar9is9hQCACH5BAADAAAALAAAAAAUABQAhGIa/2Mc/2Qd/2Ue/2gj/20r/3Ex/3U3/3c5/35D/39F/4RM/4ZO/4pV/5Zm/6mC/7SS/7eW/7iY/7qa/7qb/7yd/+TY/+XZ/+ba/+fc/+je/+nf/////wAAAAAAAAAAAAUyICeOZGmeaKqubOu+MHtFUvZeCSAsmwsVCMeh4jM8GgJi66IgBHg3iaQXq1qv2Kz2FAIAOw==) `;
```

Comment:
I know that this was not introduced by this PR, but could you please move it to a file with constants?